pkg/server/domain/model/application.go

```diff
@@ -39,6 +39,7 @@ type Application struct {
 	Description string            `json:"description"`
 	Icon        string            `json:"icon"`
 	Labels      map[string]string `json:"labels,omitempty"`
+	Annotations map[string]string `json:"annotations,omitempty"`
```

Comment:
Let’s assign the annotations to application CR when deploying it. And make sure don’t override the inner annotations.

Reply:
So need to consider annotations when creating applications? Does 'inner annotations' refer to the 'annotations' attribute of CR metadata?

Reply:
Now, you only added the annotation field for the application DB model. After you deploy an application, you can use the `vela` CLI command to view the application CR. It is a real request feature that we could find the annotations in the application CR. Do you think so?

What is the inner annotation? Like the `app.oam.dev/publishVersion` is an inner annotation. It is generated by the system.

Reply:
@yikuaibro